doc/LanguageClient.txt

```diff
@@ -281,6 +281,26 @@ neovim >= 0.3.0.
 Default: 0
 Valid options: 1 | 0
 
+2.22 g:LanguageClient_documentHighlightDisplay *g:LanguageClient_documentHighlightDisplay*
+
+Control how document highlights are displayed.
+
+Default: >
+    {
+        1: {
+            "name": "Text",
+            "texthl": "LanguageClientText",
+        },
+        2: {
+            "name": "Read",
+            "texthl": "LanguageClientRead",
```

Comment:
I believe with almost all theme there, there won't be such highlight groups defined. Can you change this highlight these groups to make it more useful?